source/opt/struct_cfg_analysis.cpp

```diff
@@ -150,6 +150,19 @@ uint32_t StructuredCFGAnalysis::LoopContinueBlock(uint32_t bb_id) {
   return merge_inst->GetSingleWordInOperand(kContinueNodeIndex);
 }
 
+uint32_t StructuredCFGAnalysis::LoopNestingDepth(uint32_t bb_id) {
+  uint32_t depth = 0;
```

Comment:
Up to you, but I like to use `result` for the name of the thing that will ultimately be returned by a method.

Reply:
Done (also in #3754 )